Makefile

```diff
@@ -212,35 +225,44 @@ dashboard/assets.go: $(DASHBOARDSOURCES) $(DASHBOARDDIR)/Dockerfile.build
 		$(DASHBOARDBUILDIMAGE)
 	$(GOASSETSBUILDER) -s /dashboard/build/ -o dashboard/assets.go -p dashboard dashboard/build
 
-$(BIN): $(GOBUILDDIR) $(CACHEVOL) $(SOURCES) dashboard/assets.go
+.PHONY: build-bin
+build-bin: $(GOX) $(GOBUILDDIR) $(CACHEVOL) $(SOURCES) dashboard/assets.go
 	@mkdir -p $(BINDIR)
-	docker run \
-		--rm \
-		-v $(SRCDIR):/usr/code \
-		-v $(CACHEVOL):/usr/gocache \
-		-e GOCACHE=/usr/gocache \
-		-e GOPATH=/usr/code/.gobuild \
-		-e GOOS=linux \
-		-e GOARCH=amd64 \
-		-e CGO_ENABLED=0 \
-		-w /usr/code/ \
-		golang:$(GOVERSION) \
-		go build -installsuffix cgo -ldflags "-X main.projectVersion=$(VERSION) -X main.projectBuild=$(COMMIT)" -o /usr/code/bin/$(BINNAME) $(REPOPATH)
+	CGO_ENABLED=0 GOPATH=$(GOBUILDDIR) $(GOX) \
+		-os="linux" \
+		-arch="$(ARCHS)" \
+		-osarch="!darwin/arm !darwin/arm64" \
+		-ldflags="-X main.projectVersion=${VERSION} -X main.projectBuild=${COMMIT}" \
+		-output="bin/{{.OS}}/{{.Arch}}/$(BINNAME)" \
+		-tags="netgo" \
+		github.com/arangodb/kube-arangodb
+	@ln -sf $(BINDIR)/$(shell go env GOOS)/$(shell go env GOARCH)/kube-arangodb$(shell go env GOEXE)
 
 
 
 .PHONY: docker
-docker: check-vars $(BIN)
-	docker build -f $(DOCKERFILE) -t $(OPERATORIMAGE) .
-ifdef PUSHIMAGES
-	docker push $(OPERATORIMAGE)
-endif
+docker: check-vars build-bin $(MANIFESTTOOL)
+	for arch in $(ARCHS); do \
+		docker build --build-arg=GOARCH=$$arch -t $(OPERATORIMAGE)-$$arch . ;\
+		docker push $(OPERATORIMAGE)-$$arch ;\
+	done
+	$(MANIFESTTOOL) $(MANIFESTAUTH) push from-args \
```

Comment:
This does not run (out of the box) on macos.
The problem is docker registry authentication. That's why `MANIFESTAUTH` environment variable is in there.
It has to be set to something like `--username=foo --password=bar`.

See https://github.com/estesp/manifest-tool#sample-usage

Suggest to add this to the README in the building section.